build.gradle

```diff
@@ -180,19 +175,17 @@ task pushDockerImage {
 
 task removeOldDockerContainer(type: Exec) {
     commandLine "bash", "-c",
-            "for DC in `docker ps -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker stop \$DC; done && " +
-            "for DC in `docker ps -a -f 'name=$dockerContainerName' | grep -o $dockerContainerName`; do docker rm \$DC; done"
+            "docker ps -aqf name=$dockerContainerName | xargs -r docker rm -f"
```

Comment:
Unfortunately, the `-r` option is not in POSIX compliant and it's failing on a Mac. Sad story, but that's why I did this `for` hazzle. :(